packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -265,12 +328,13 @@ function mountVNodes(
     parent: ParentNode,
     anchor: Node | null,
     start: number = 0,
-    end: number = vnodes.length
+    end: number = vnodes.length,
+    vm: VM
```

Comment:
The `vm` argument should be moved prior to the `start` argument. Otherwise, there is no point in having default values for `start` and `end`.